index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Kalolo's Portfolio</title>
+</head>
+
+<body>
+  <header class="white">
+    <nav class="nav-bar">
+
+      <div class="logo">
+        <a href="#">My logo</a>
+      </div>
+      <div class="menu-icon"><img src="images/menu.png" alt="menu navbar"></div>
+    </nav>
+  </header>
+    <section class="first-section">
+      <div class="text">
+        <h1 class="salutation">
+          I'm Kalolo, glad to see you!
+        </h1>
+        <p class="paragraph1">I’m a software developer! I can help you build a product , feature or website Look through
+          some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+          contact me.</p>
+
+        <p class="lets-connect"> LET'S CONNECT</p>
+
+        <ul class="buttons" >
+          <li><a href="#"><img src="./images/twitter.png" alt="social"> </a></li>
+          <li><a href="#"><img src="./images/Linkedin.png" alt="social"></a></li>
+          <li><a href="#"><img src="./images/medium.png" alt="social"></a></li>
+          <li><a href="#"><img src="./images/github.png" alt="social"></a></li>
+          <li><a href="#"><img src="./images/angellist.png" alt="social"></a></li>
+        </ul>
+      </div>
+    </section>
```

Comment:
- [ ] **[REQUIRED CHANGES]** I would suggest you add this `section` to be part of the `header` of your application. This is important for SEOs to better understand what kind of information to display on Searching Pages (i.e google, duck duck go).
Remember that the Header of the page is how you present yourself to the internet, so it would be great to have your name and a little bit about you and your work, as we have on the `first-section` (which is looking great by the way :fire: )!

- [ ] **[REQUIRED CHANGES]** Could you change the `first-section` to contain the padding provided by the template? Which would be `120px 24px`. You can see it on the right side of the Figma page, on clicking on an element:
![image](https://user-images.githubusercontent.com/47700024/137761568-6933c60b-ac19-4dab-a8a7-737a53ff217f.png)


- [ ] **[OPTIONAL]** Try to think of more meaningful names for your classes, so this way, you can also use them to inform other programmers what they are responsible for, as you did for the `salutation` class. A suggestion for you would be the following:
```html
<p class="about-me">I'm a software developer...</p>
```